### PR TITLE
What: Updated process for proposals to projects

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* lack of clarity and specific steps in previous process introduced confusion.

This change addresses the need by:

* clarifying the process #609
* describing the issue types
* updating the proposal template
* clearly stating OWNERS of projects as requested in #487

### CODE REVIEW

File: governance/process.md
Comment (ultrasaurus) [reply governance/process.md:68]: I tried to make it clear below that discussion via slack/email/etc is welcome.  The more explicit this is, though, the harder it is to expect people to read the whole thing, so we should also balance concise instructions.

File: governance/process.md
Comment (ultrasaurus) [inline governance/process.md:77]: again, let's keep meetings optional, can add "if they can" but the planned meeting should be noted on the issue and we should steer people to always update the issue -- things work well when everyone looks to the info in the issue as the source of truth, so it's friendly to new participation and leadership team or other project leaders and other groups can also look to the issue to see what is going on

File: governance/process.md
Comment (ultrasaurus) [inline governance/process.md:82]:
```suggestion
   B) Choose an upcoming meeting where you or another
```
let's not encode two regions!    We may someday split Europe/Africa from Americas

File: governance/process.md
Comment (ultrasaurus) [inline governance/process.md:80]: I dunno -- if no one is interested in sponsoring it, which typically happens if there's too much going on right now, do we want to encourage scoping/investment in getting the thing done?  In practice, I've rarely seen something well scoped without the collaboration of a Chair or TL

File: governance/process.md
Comment (ultrasaurus) [inline governance/process.md:146]: When originally written, this option was considered way earlier in the process.  At any time, the proposal may be closed by leadership team as out of scope of the group, but we agreed not to do that before it was discussed at a meeting (hence original order) but it we put scoping first then this doesn't feel right here.

**Review by ultrasaurus — COMMENTED** [review]
Thanks for pulling this together!  I made a bunch of suggested edits, tried to annotate which I feel are required and which are just ideas.

The main issue that probably needs more of an edit is about closing the proposal and how/when that happens.

Also, minor point: next time, would be nice to avoid gratuitous changes in word-wrapping. It requires a lot of visual diffing, since that doesn't get screened out by github as whitespace diffs.

File: .github/ISSUE_TEMPLATE/proposal.md
Comment (TheFoxAtWork) [inline .github/ISSUE_TEMPLATE/proposal.md:20]:
```suggestion
```

Comment (TheFoxAtWork) [reply .github/ISSUE_TEMPLATE/proposal.md:20]: good catch - i thought i had removed this.

File: governance/process.md
Comment (TheFoxAtWork) [reply governance/process.md:57]: I think it depends.  the general channel (to me) seems like the go-to place for help or posting things.  governance does as well.  triage has a fair amount of traffic but is really the right place for that notice - or governance?

File: governance/process.md
Comment (TheFoxAtWork) [inline governance/process.md:58]:
```suggestion
   channel to bring attention to it. Skip to ask for collaboration.
```

File: governance/process.md
Comment (TheFoxAtWork) [inline governance/process.md:77]:
```suggestion
   the problem, if so, refer them to the issue which has content on discussions/meetings
```

File: governance/process.md
Comment (TheFoxAtWork) [reply governance/process.md:80]: IME - scoping is driven my the group or author with feedback from the Chair TL.  Doesn't mean it doesn't happen, and ultimately those conversations that drive scoping are usually with the chair/TL that is willing to sponsor (as sponsor they "sign off" on the scope as well?).  I've not seen where we don't sponsor due to timing, rather we sponsor but postpone the work.

File: governance/process.md
Comment (TheFoxAtWork) [reply governance/process.md:146]: A lot of suggestions or proposals that come to the group arent fully formed.  Scoping allows us to be more definitive in what is or is not aligned with the charter.

File: governance/process.md
Comment (ultrasaurus) [reply governance/process.md:80]: There are currently 21 [open proposals](https://github.com/cncf/tag-security/issues?q=is%3Aissue+is%3Aopen+label%3Aproposal) and I thought most were open due to lack of bandwidth. (If TL/Chairs weren't already working on so many things in parallel, then I figured some of those would be moving forward.)

File: governance/process.md
Comment (ultrasaurus) [reply governance/process.md:146]: yes, agreed for something that is clearly within our charter.  If something comes to us that is not appropriate to the group, then we want to close early in the process.

File: governance/process.md
Comment (TheFoxAtWork) [reply governance/process.md:80]: I think a lot of these no longer have interest.  its worth the time to sort through them and triage them accordingly.  thanks for bringing this up.

File: governance/process.md
Comment (lumjjb) [inline governance/process.md:200]: I'm a little unsure about the voting system, I think we haven't had a situation where this is needed yet, and wondering if it would be best to hold off on this until we see the need to... I think leaving it to the decision of leadership (or a unanimous decision among leadership) would be good enough.

Comment (lumjjb) [reply governance/process.md:200]: If we do decide to go with voting, I think that we need to have a bit more "system" around this. Voting needs to ensure visibility of the vote happening, maybe via email list. There needs to be proof of participation provided with the vote so we don't have to go around digging up history. We need to ensure that there isn't bad faith with editing meeting notes to include past participation since there is now an incentive to do so.

File: governance/roles.md
Comment (lumjjb) [inline governance/roles.md:51]: it's not quite clear here whether this is refering to our generic member list of members of a PROJECT role. i.e. removing from README or removing from issue/PR.

I don't think that there should be removal of folks from the README (but perhaps an active/inactive bit), since some that have made big contributions in the past but are no longer active should still be recognized as members.

File: governance/roles.md
Comment (lumjjb) [inline governance/roles.md:104]: add link to new role description

File: governance/process.md
Comment (ashutosh-narkar) [inline governance/process.md:193]: Nit: `not` -> `no`

File: governance/process.md
Comment (ashutosh-narkar) [reply governance/process.md:200]: I agree with Brandon that the final decision should be left with the leadership team. We could do a majority or 2/3rd vote in the leadership team. We can encourage community members to submit "non-binding" votes to show support or lack of it.

File: governance/process.md
Comment (TheFoxAtWork) [inline governance/process.md:193]:
```suggestion
   vote is not required. If there’s no consensus among the group, a formal vote
```

File: governance/roles.md
Comment (lumjjb) [reply governance/roles.md:51]: reading this over again, i think the Role specific clarifies this

**Comment by lumjjb** [comment]
@TheFoxAtWork can we make the 1 added link to Chair Emeritus, and i think we can merge this in?

File: governance/roles.md
Comment (TheFoxAtWork) [inline governance/roles.md:103]:
```suggestion
role of [Chair Emeritus](#role-of-chair-emeriti). This allows previous Chairs to continue to chime in and
```

File: governance/roles.md
Comment (TheFoxAtWork) [reply governance/roles.md:104]:
```suggestion
provide valuable context and contributions to the TAG. A [Chair Emeritus](#role-of-chair-emeriti) can
```

**Comment by lumjjb** [comment]
strange, why is the CI failing now when previously it didnt.

File: governance/process.md
Comment (TheFoxAtWork) [inline governance/process.md:186]:
```suggestion
   the next meeting. If there are no objections, or if all concerns have been
```

**Comment by TheFoxAtWork** [comment]
Okay i think i have fixed EVERYTHING and added in the new issue on STAG reps v project leads. #774  
@lumjjb @ashutosh-narkar @anvega @achetal01 please review!!

File: governance/process.md
Comment (ashutosh-narkar) [inline governance/process.md:78]: Nit: `provide` -> `Provide`

File: governance/process.md
Comment (ashutosh-narkar) [inline governance/process.md:93]: Nit: `of` -> `if`

**Review by ashutosh-narkar — COMMENTED** [review]
Apart from the voting mechanism on the PRs the rest lgtm. Regarding the voting, imo we could consider something like binding and non-binding votes. The STAG leadership has the binding votes. I think the current approach could be difficult to keep track off.

**Comment by TheFoxAtWork** [comment]
@ashutosh-narkar all done!  please review and merge!

**Review by anvega — APPROVED** [review]

**Review by anvega — APPROVED** [review]

**Pull request merged**
